common/djangoapps/third_party_auth/provider.py

```diff
@@ -52,6 +52,11 @@ def get(cls, provider_id):
             return None
 
     @classmethod
+    def get_all_provider_ids(cls):
+        """ Gets a list of all enabled provider ids """
+        return [provider.provider_id for provider in cls._enabled_providers()]
```

Comment:
Please remove this method (`get_all_provider_ids`) and its test as well, if it's unused.